h/claim/invite.py

```diff
@@ -0,0 +1,182 @@
+# -*- coding: utf-8 -*-
+"""
+:mod:`h.claim.invite` is a utility to invite users to claim accounts
+and is exposed as the command-line utility hypothesis-invite.
+"""
+import argparse
+import logging
+import os
+import sys
+import time
+
+import mandrill
+from pyramid import paster
+from pyramid.request import Request
+import transaction
+
+from h.accounts import models
+from h.claim.util import generate_claim_url
+
+log = logging.getLogger('h.claim.invite')
+
+
+def get_env(config_uri):
+    """Return a preconfigured paste environment object."""
+    env = paster.bootstrap(config_uri)
+    return env
+
+parser = argparse.ArgumentParser(
+    'hypothesis-invite',
+    description='Send invitation emails to users.'
+)
+parser.add_argument('config_uri', help='paster configuration URI')
+parser.add_argument(
+    '--base',
+    help='base URL',
+    default='http://localhost:5000',
+    metavar='URL'
+)
+parser.add_argument(
+    '-n',
+    '--dry-run',
+    help='dry run (log but do not send email)',
+    action='store_true',
+)
+parser.add_argument(
+    '-l',
+    '--limit',
+    type=int,
+    metavar='N',
+    help='maximum users to invite',
+)
+parser.add_argument(
+    '-k',
+    '--key',
+    metavar='KEY',
+    help='Mandrill API key (defaults to MANDRILL_APIKEY variable)',
+    default=os.environ.get('MANDRILL_APIKEY'),
+)
+
+
+def get_users(session, limit=None):
+    return session \
+        .query(models.User) \
+        .filter(
+            models.User.password == u'',
+            models.User.status.op('&')(0b1000) == 0  # noqa
+        ) \
+        .limit(limit) \
+        .all()
+
+
+def get_merge_vars(request, users):
+    for user in users:
+        claim = generate_claim_url(request, str(user.id))
```

Comment:
Does this work? I think `generate_claim_url` is [*intended* to take acct URIs](https://github.com/hypothesis/h/blob/d488f9c491c0334ee4f46210ba17127df169a9e6/h/claim/test/utils_test.py#L16)?